atomic_reactor/util.py

```diff
@@ -1819,3 +1819,12 @@ def _enclose_pullable(self):
 
             if self._should_enclose(image):
                 image.enclose(self._organization)
+
+
+def sha256sum(s, abbrev_len=0, prefix=False):
+    prefix_ = 'sha256:' if prefix else ''
+    data = s.encode() if type(s) == str else s
```

Comment:
@tkdchen; To satisfy my own curiosity - aren't Py 3 strings unicode by default?

Reply:
@ben-alkov This is refactored from `make_digest`, which calls `to_bytes` to accept either a str or bytes string and ensure a bytes string is passed to `hashlib.sha256`.